raiden/network/blockchain_service.py

```diff
@@ -102,8 +102,11 @@ def get_block_header(self, block_number: int):
         return self.client.web3.eth.getBlock(block_number, False)
 
     def next_block(self) -> int:
+        target_block_number = self.block_number() + 1
+        self.wait_until_block(target_block_number=target_block_number)
+
+    def wait_until_block(self, target_block_number):
         current_block = self.block_number()
-        target_block_number = current_block + 1
 
         while current_block < target_block_number:
             current_block = self.block_number()
```

Comment:
and bellow is the sleep used now

Reply:
Ah damn ... I had to click the "expand" button to see there was a sleep there. Thanks.